deploy/template.yaml

```diff
@@ -64,6 +64,12 @@ Conditions:
 
   EnableSpotArmInstance: !Equals [!Ref Environment, dev]
 
+  IsProductionOrIntegrationEnvironment: !Or
+    - !Equals [!Ref Environment, production]
+    - !Equals [!Ref Environment, integration]
```

Comment:
Is suppose this not needed but, since it temporary it is fine - Address-front there was an existing `IsProductionOrIntegrationEnvironment`